components/eam/src/physics/cam/aer_rad_props.F90

```diff
@@ -77,7 +79,22 @@ subroutine aer_rad_props_init()
         'EXTINCT LW H2O window band 7 output directly read from prescribed input file', flag_xyfill=.true.)
    call addfld ('extinct_sw_inp',(/ 'lev' /),    'A','1/km',&
         'Aerosol extinction directly read from prescribed input file', flag_xyfill=.true.)
-
+   ! add full bands radiation output
+   ! longwave: ext_sao_lw (pcols,pver,nlwbands)
+   ! ext_sao_sw, ssa_sao, af_sao (pcols,pver,nswbands)
+   if (is_output_interactive_volc) then
+   call add_hist_coord('nlwbands',    nlwbands,    'NLWBANDS')  !kzm
+   call add_hist_coord('nswbands',    nswbands,    'NSWBANDS')  !kzm 
+   call addfld ('ext_sao_lw',(/ 'lev', 'nlwbands' /),    'A','1/m',&
```

Comment:
For it to work across different compilers, you will want to bad `'lev'` with spaces to make `'nlwbands'` and '`lev     '` the same length. We saw the same thing with the P3 PR.